### PR TITLE
Prepare docs 1st wave of Providers December 2023 RC2

### PR DESCRIPTION
**Pull request by eladkal**
Providers for RC2:
daskexecutor, databricks, docker, google, odbc

### CODE REVIEW

File: airflow/providers/daskexecutor/CHANGELOG.rst [hunk 1]
```diff
@@ -27,11 +27,29 @@
 Changelog
 ---------
 
-1.1.1
+1.2.0
```
Comment (potiuk) [inline airflow/providers/daskexecutor/CHANGELOG.rst]: See the discussion in https://github.com/apache/airflow-site/pull/907 - I do think we should not bump neither min-airflow version nor MINOR version of the daskexecutor. 

I am open to discuss it, but I think there are good reasons why the daskexecutor  provider should be axactly the same as last released one (with added "removed" description).

Comment (eladkal) [reply airflow/providers/daskexecutor/CHANGELOG.rst]: if https://github.com/apache/airflow/pull/36191 fix the issue then yeah we should stay with 1.1.1

**Review by potiuk — APPROVED** [review]

File: airflow/providers/daskexecutor/__init__.py [hunk 2]
```diff
@@ -27,16 +27,16 @@
...
 except ImportError:
     from airflow.version import version as airflow_version
 
 if packaging.version.parse(packaging.version.parse(airflow_version).base_version) < packaging.version.parse(
-    "2.5.0"
+    "2.6.0"
 ):
     raise RuntimeError(
-        f"The package `apache-airflow-providers-daskexecutor:{__version__}` needs Apache Airflow 2.5.0+"
+        f"The package `apache-airflow-providers-daskexecutor:{__version__}` needs Apache Airflow 2.6.0+"
```
Comment (eladkal) [inline airflow/providers/daskexecutor/__init__.py]: also needs to be reverted back

Comment (potiuk) [reply airflow/providers/daskexecutor/__init__.py]: Yep. Just added fixup and go through it again.

**Pull request merged**